packages/now-next/src/index.ts

```diff
@@ -432,6 +439,26 @@ export const build = async ({
   let dynamicRoutes: Route[] = [];
   // whether they have enabled pages/404.js as the custom 404 page
   let hasPages404 = false;
+  let buildId = '';
+  let escapedBuildId = '';
+
+  if (isLegacy || isSharedLambdas) {
+    try {
+      buildId = await readFile(
+        path.join(entryPath, outputDirectory, 'BUILD_ID'),
+        'utf8'
+      );
+      escapedBuildId = escapeStringRegexp(buildId);
+    } catch (err) {
+      console.error(
+        'BUILD_ID not found in ".next". The "package.json" "build" script did not run "next build"'
+      );
+      throw new NowBuildError({
+        code: 'NOW_NEXT_NO_BUILD_ID',
+        message: 'Missing BUILD_ID',
```

Comment:
```suggestion
        message: 'The BUILD_ID file was not found in the Output Directory. Did you forget to run "next build" in your Build Command?',
```

We need to make this actionable because its what the user sees. I dont think we need the console.error() though